### ISSUE TITLE
Changes required for upcoming CoffeeScript upgrade

### ISSUE THREAD

**Issue by maxbrunsfeld** [gstack/atom-browser-webview#33]
Hi! Thanks for maintaining the atom-webbrowser package!

In Atom v1.12, we are going to [upgrade CoffeeScript](https://github.com/atom/atom/pull/12780) to the latest version. This upgrade entails one potentially breaking change to the language:

> Changed strategy for the generation of internal compiler variable names. Note that this means that @example function parameters are no longer available as naked example variables within the function body.

We think your package may be affected by this upgrade, in the following places:
- The `browser` variable [here](https://github.com/gstack/atom-browser-webview/tree/master/lib/toolbar-view.coffee#L33)

These findings are based on linting packages with `coffeescope`. We could be wrong about some of them. When we release v1.12 beta, please test your package against it to make sure that it works. Let me know if you have any further questions; I will be happy to help!

**Comment by BillyDeeWilliams** [comment]
I am encountering this error. I a  student and new to github, and git in general. is there a work around for this?

**Comment by maxbrunsfeld** [comment]
Yeah, in your local copy of `atom-browser-webview` (probably located in `~/.atom/packages/atom-browser-webview`) you can replace `browser` with `@browser` on line 33 of `lib/toolbar-view.coffee` (the line linked above).

**Comment by btquanto** [comment]
Apparently, that alone is not enough.
